config/space-presets.json

```diff
@@ -48,7 +48,7 @@
     }
   },
   {
-    "id": "5apps",
+    "id": "xmpp-5apps",
```

Comment:
I changed the code to not use a computed property for the id anymore. Now it's only generating an id from the name when no explicit id was given. This way everything works fine now.